Link errors when building on linux

I get the following errors when building the current git code using the v2014.1 compiler.

```
LinkEXE /tmp/opendylan/Bootstrap.2/bin/dylan-compiler : _main.c
/tmp/opendylan/Bootstrap.2/lib/libdylan.so: undefined reference to `TYPE_CHECK_ARGS_8'
/tmp/opendylan/Bootstrap.2/lib/libdylan.so: undefined reference to `TYPE_CHECK_ARGS_7'
/tmp/opendylan/Bootstrap.2/lib/libdylan.so: undefined reference to `TYPE_CHECK_ARGS_5'
/tmp/opendylan/Bootstrap.2/lib/libdylan.so: undefined reference to `process_keyword_call_into'
/tmp/opendylan/Bootstrap.2/lib/libdylan.so: undefined reference to `TYPE_CHECK_ARGS_9'
/tmp/opendylan/Bootstrap.2/lib/libdylan.so: undefined reference to `TYPE_CHECK_ARGS_6'

collect2: error: ld returned 1 exit status
```

Linux Lubuntu 17.10 64-bit, gcc 7.2

--- a/sources/lib/run-time/c-run-time.c
+++ b/sources/lib/run-time/c-run-time.c
@@ -58,7 +58,7 @@ extern dylan_object KPunboundVKi;
 #define INLINE static inline
 #define FORCE_INLINE static inline __attribute__((always_inline))
 #else
-#define INLINE inline
+#define INLINE static inline
 #define FORCE_INLINE inline __attribute__((always_inline))
 #endif
 

